src/main/java/dbaccessor/message/MessageAccessor.java

```diff
@@ -34,6 +39,36 @@ public void updateReceivedTs(Message message) {
         spannerTemplate.update(message, "MessageID", "ReceivedTS");
     }
 
+    /**
+     * Completes all DB insertions for the CreateMessage API in a single transaction.
+     */
+    public boolean insertForCreateMessageTransaction(Message message, Chat chat) {
```

Comment:
Why not call it `createMessageInTransaction`?
That sounds like a cleaner name to me.

Reply:
+1

Reply:
Done